src/test/java/com/getindata/connectors/http/internal/status/HttpCodesParserTest.java

```diff
@@ -0,0 +1,91 @@
+package com.getindata.connectors.http.internal.status;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Singular;
+import org.apache.flink.util.ConfigurationException;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class HttpCodesParserTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = {
+            "6XX",
+            "4xx",
```

Comment:
nit: I think we can accept `4xx` by adding uppercase() before processing configuration.